staff/gerard-celma/e-commerce-template/index.htm

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>e-commerce</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+</head>
+<body>
+    <header>
+        <div>
+            <i class="fab fa-google fa-3x"></i>
+            <p id="logo-name">Google</p>
+        </div>
+        <form action="#">
+            <input type="text" name="" id="" placeholder="type here">
+            <i class="fas fa-search"></i>
+        </form>
+        <i class="fas fa-bars fa-2x" id="burger"></i>
+        <div id="menu">
+            <p>about</p>
+            <p>contact</p>
+        </div>
```

Comment:
To have an organic webside, instead of using `<div>` I would use `<nav>` and each link `<a>` to put the link.